### DISCUSSION TITLE
CPU frequency

### DISCUSSION THREAD

**Discussion by Orsso** [seerge/g-helper#1529] [Q&A]
Hi folks ! Thank you for this amazing software ! I was wondering if there is a way to limit CPU freqeuncies for each profile ? I would like to make a "ultrabook" mode on my G14 that would let me get rid of active cooling. I've already limit TDP to 20w which fit great for bureautic tasks. My version is R7 4800HS.

**Comment by seerge** [comment]
@Orsso hello, I don't plan to add that as it's too custom and specific :) but you can limit frequencies inside windws power plan settings 

https://www.majorgeeks.com/content/page/how_to_add_or_remove_maximum_processor_frequency.html

**Reply by Orsso** [reply]
thank you very much for the quick answer it helped a lot
